### ISSUE TITLE
IE11 Javascript error on Smooch.destroy()

### ISSUE THREAD

**Issue by GeorgeStevensLhhp** [smooch/smooch-web#661]
**For support questions, please visit [our help page](https://smooch.io/help/)**.

This repository's issues are reserved for bug reports only.

 **I'm submitting a ...**
  - [x] bug report
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

 **What is the current behavior?**
On IE11, a javascript error is seen when ending a chat with Smooch.destroy() 
```
TypeError: Object doesn't support property or method 'remove'
   at _.destroy (https://cdn.smooch.io/smooch.4.8.2.min.js:5:4597)
```
This also prevents our javascript from re-initiating the chat window if the user wants to start a new session afterwards.

 **Please provide the steps to reproduce and, if possible, a minimal demo of the problem**
Open developer tools in IE.
Open Smooch chat session with Smooch.Init().
call Smooch.Destroy() to end chat session.

 **What is the expected behavior?**
The chat window closes without throwing any errors 

 **Please tell us about your environment:**
- Delivery method: CDN 
- Version: 4.8.2
- Browser: IE 11
- OS: Windows 7

 **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. Stack Overflow, etc)

**Comment by mspensieri** [comment]
Thanks for the report. Does this only happen on IE 11, or other browsers as well?

**Comment by GeorgeStevensLhhp** [comment]
We have only seen this issue on IE11, it works fine on Chrome and Firefox.

**Comment by mspensieri** [comment]
Should be fixed in [v4.8.4](https://github.com/smooch/smooch-web/releases/tag/4.8.4)
